jsettlers.main.swing/src/main/java/jsettlers/main/swing/resources/SettlersFolderChecker.java

```diff
@@ -64,13 +62,16 @@ public static SettlersFolderInfo checkSettlersFolder(String settlersBaseFolder)
 
 		for (File f : files) {
 			if (f.getName().equalsIgnoreCase("snd")) {
-				File testSndFile = new File(f, "Siedler3_00.dat");
-				if (testSndFile.exists()) {
-					sndFolder = f;
+				for (File g : f.listFiles()) {
+					g.getName();
+					if (g.getName().equalsIgnoreCase("siedler3_00.dat")) {
```

Comment:
Again, switch the argument and the object to 
`"siedler3_00.dat".equalsIgnoreCase(g.getName())`